lib/server.js

```diff
@@ -866,13 +887,10 @@ WalletService.prototype.removePendingTx = function(opts, cb) {
         return cb(new ClientError('TXNOTPENDING', 'Transaction proposal not pending'));
 
 
-      if (txp.creatorId !== self.copayerId)
-        return cb(new ClientError('Only creators can remove pending proposals'));
-
-      var actors = txp.getActors();
-
-      if (actors.length > 1 || (actors.length == 1 && actors[0] !== self.copayerId))
-        return cb(new ClientError('TXACTIONED', 'Cannot remove a proposal signed/rejected by other copayers'));
+      var deleteLockTime = self.getRemainingDeleteLockTime(txp);
+      if (deleteLockTime > 0) {
+        return cb(new ClientError('TXCANNOTREMOVE', 'Cannot remove this tx proposal during locktime. Time remaining:' + deleteLockTime));
```

Comment:
I don't think we need a code for this error. I would just send the raw error message since having a code means we need to add this to the translations in the UX. Do you agree?